src/neo/Network/P2P/Payloads/GetBlockByIndexPayload.cs

```diff
@@ -50,7 +50,7 @@ void ISerializable.Deserialize(ref MemoryReader reader)
         {
             IndexStart = reader.ReadUInt32();
             Count = reader.ReadInt16();
-            if (Count < -1 || Count == 0 || Count > HeadersPayload.MaxHeadersCount)
+            if (Count is < -1 or 0 or > HeadersPayload.MaxHeadersCount)
```

Comment:
I prefer the original version

Reply:
Agree. Can you revert this kind of changes? @Liaojinghui

Reply:
We don't currently have strict code style requirements, so I don't think it's necessary to change this at the moment.

Reply:
~~~ has reverted on those typo-unrelated changes.